tests/pyspark_delta/test_pyspark_delta.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import chispa
+import pyspark
+from delta import *
+
+builder = (
+    pyspark.sql.SparkSession.builder.appName("MyApp")
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+    .config(
+        "spark.sql.catalog.spark_catalog",
+        "org.apache.spark.sql.delta.catalog.DeltaCatalog",
+    )
+)
+
+spark = configure_spark_with_delta_pip(builder).getOrCreate()
+
+def test_reference_table_1_content():
+    versions = [0, 1]
+    for v in versions:
+        actual_df = spark.read.format("delta").option("versionAsOf", v).load("out/tables/generated/reference_table_1/delta")
+        expected_df = spark.read.format("parquet").load(f"out/tables/generated/reference_table_1/expected/v{v}/table_content.parquet")
+        chispa.assert_df_equality(actual_df, expected_df)
+
+
+def test_reference_table_2_content():
```

Comment:
Instead of writing a test function for each case, we should be able to write code that will collect all the test cases and them pass them into one parameterized case. See what I did in: https://github.com/delta-io/delta-rs/pull/909/files